tests/robot-tests/tests/libs/common.robot

```diff
@@ -773,13 +773,13 @@ user closes Set Page View box
     user waits until element is not visible    id:editingMode
 
 user checks page for details section
-    [Arguments]    ${heading}
-    user checks page contains element    testid:Expand Details Section ${heading}
```

Comment:
In `common.robot`, `user checks page contains element` doesn't take a wait argument, so this keyword is currently going to always fail due to having the wrong number of arguments.

Same for `user checks page for details dropdown`.